StaticAnalyzer/views/android/VirusTotal.py

```diff
@@ -51,7 +60,16 @@ def upload_file(self, file_path):
                 "apikey": settings.VT_API_KEY
             }
             try:
-                response = requests.post(url, files=files, data=headers)
+                if settings.UPSTREAM_PROXY_ENABLED:
+                    if not settings.UPSTREAM_PROXY_USERNAME:
+                        proxy_port = str(settings.UPSTREAM_PROXY_PORT)

```

Comment:
Why repeating this `proxy_port = str(settings.UPSTREAM_PROXY_PORT)` put this above `if not settings.UPSTREAM_PROXY_USERNAME` and it will be in scope within both if and else.

Reply:
will do :)